docs/HOWTO/how_to_build_library_to_encapsulate_openvino.md

```diff
@@ -0,0 +1,100 @@
+# How to build library to encapsulate OpenVINO
+
+> **NOTE**: This guide illustrated a simplified example about how to build a customer library based on OpenVINO.
```

Comment:
why is it needed? We have https://docs.openvino.ai/latest/openvino_docs_OV_UG_Integrate_OV_with_your_application.html#link-and-build-your-c-application-with-openvino-runtime

Reply:
A difference is one is to build into application and one is to build into a shared library.  Though they are only tiny different, but some customer don't know how to do it. So we should add this part.
Anyhow, it should be better to put this part into https://docs.openvino.ai/latest/openvino_docs_OV_UG_Integrate_OV_with_your_application.html#link-and-build-your-c-application-with-openvino-runtime ?